addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -3253,25 +3252,24 @@ export class OdooEditor extends EventTarget {
                     selection &&
                     selection.anchorNode &&
                     !closestElement(selection.anchorNode).closest('a') &&
-                    selection.anchorNode.nodeType === Node.TEXT_NODE &&
-                    !this.powerbox.isOpen
+                    selection.anchorNode.nodeType === Node.TEXT_NODE
                 ) {
                     const textSliced = selection.anchorNode.textContent.slice(0, selection.anchorOffset);
                     const textNodeSplitted = textSliced.split(/\s/);
 
                     // Remove added space
                     textNodeSplitted.pop();
                     const potentialUrl = textNodeSplitted.pop();
-                    const lastWordMatch = potentialUrl.match(URL_REGEX_WITH_INFOS);
-
-                    if (lastWordMatch) {
-                        const matches = getUrlsInfosInString(textSliced);
-                        const match = matches[matches.length - 1];
+                    const match = potentialUrl.match(HTTP_REGEX);
```

Comment:
I see that this uses the new regex. What was the problem with the previous one ? Mainly the @ I guess ? Wouldn't it be possible to design one regex that is sufficient for both cases without needing two different regexes with very similar names ? If _no_ but the only meaningful difference is whether a `@` is allowed or not, then maybe we could just test if the string includes `@` rather than have yet another http regex ? My point is to try to see if it's possible to have only one regex for both cases.

Reply:
Got rid of HTTP_REGEX.